src/Symfony/Bridge/Doctrine/Translation/DoctrineMessageCatalogue.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Translation;
+
+use Doctrine\Common\Cache\Cache;
+use Doctrine\Common\Cache\MultiGetCache;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+class DoctrineMessageCatalogue extends MessageCatalogue
+{
+    const PREFIX = 'sf2_translation';
```

Comment:
no need for prefixes. doctrine cache has so called namespaces which people can use to avoid collisions.